web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -696,30 +824,44 @@ const TeamReviews = ({ onBack, periodId }) => {
                 <Delete />
               </IconButton>
             </Tooltip>
+            {approvalMode && (
+              <FormControlLabel
+                control={
+                  <Switch
+                    checked={showAll}
+                    onChange={() => setShowAll(b => !b)}
+                  />
+                }
+                label="Show All"
+                sx={{ marginLeft: '0.5rem' }}
+              />
+            )}
           </div>
         )}
       </div>
       {period && (
-        <div className="datePickerFlexWrapper">
```

Comment:
The `className` on this wrapper `div` was changed to follow typical CSS naming conventions.